docs/reference/glossary.rst

```diff
@@ -0,0 +1,73 @@
+========
+Glossary
+========
+
+.. contents::
+   :local:
+   :backlinks: none
+
+.. We're not using the .. glossary:: admonition here, as we want to split the glossary terms into sections, which the tag does not support. If this ever changes, then we can add the tagging.
+
+Product Features
+================ 
+
+.. note:: All Flocker documentation will refer the `Production ready` product, unless tagged `Alpha` or `Beta`.
+
+Production Ready
+   This product or feature is suitable for production use. You can expect API stability between versions, robust security, and product stability for all supported uses.
+
+Beta
+   This product or feature is approaching readiness. You can expect minor API instability between versions, incomplete security, and product stability for most common uses.
+
+Alpha
+   This product or feature is a proof-of-concept. You can expect API instability, lack of security, and product instability.
+
+Flocker Terms
+=============
+
+Flocker API
```

Comment:
Elsewhere we've been calling it the "Flocker REST API" - it is confusing to have a glossary which doesn't exactly match the text it describes.